packages/pages/src/util/fetch.ts

```diff
@@ -0,0 +1,18 @@
+import { getRuntime } from "./runtime";
+
+const fetchInternal = async (
+  input: RequestInfo | URL,
+  init?: RequestInit | undefined
+): Promise<any> => {
+  const runtime = getRuntime();
+
+  // Check if running in Node 17 or less, which does not support native fetch
+  if (runtime.name == "node" && runtime.getNodeMajorVersion() < 18) {
+    const { default: crossFetch } = await import("cross-fetch");
+    return crossFetch(input, init);
+  }
+
+  return fetch(input, init);
+};
+
+export { fetchInternal as fetch };
```

Comment:
Add documentation since this component is exported publicly

Reply:
same for runtime

Reply:
We will probably also want at least a readme for this package as well since it'd otherwise be undocumented outside of the starter